azdev/commands.py

```diff
@@ -27,6 +27,9 @@ def operation_group(name):
     with CommandGroup(self, '', operation_group('linter')) as g:
         g.command('linter', 'run_linter')
 
+    with CommandGroup(self, 'test-coverage', operation_group('test_coverage')) as g:
```

Comment:
How about change to `coverage` ?

```suggestion
    with CommandGroup(self, 'coverage', operation_group('test_coverage')) as g:
```